ci/scripts/driver.sh

```diff
@@ -121,27 +118,42 @@ for pr in ${pr_list}; do
     if [[ ! -L pygw ]]; then
       ln -s "${HOMEGFS}/ush/python/pygw/src/pygw" pygw
     fi
-    PSLOT=C96C48_hybatmDA # TODO loop over experments for each yaml file in the experiments dir
-    "${WF_ROOT_DIR}/ci/scripts/create_experiment.py" --yaml "${WF_ROOT_DIR}/ci/experiments/${PSLOT}.yaml"
-    ci_status=$?
-    if [[ ${ci_status} -eq 0 ]]; then
-      {
-        echo "Created experiment"
-        echo "Experiment setup: Completed at $(date) for expirment ${PSLOT}" || true
-      } >> "${GFS_CI_ROOT}/PR/${pr}/output_${id}"
-      "${GH}" pr comment "${pr}" --repo "${repo_url}" --body-file "${GFS_CI_ROOT}/PR/${pr}/output_${id}"
-      "${GH}" pr edit --repo "${repo_url}" "${pr}" --remove-label "${CI_HOST}-Running" --add-label "${CI_HOST}-Passed"
-    else
-      "${GH}" pr edit "${pr}" --repo "${repo_url}" --remove-label "${CI_HOST}-Running" --add-label "${CI_HOST}-Failed"
-    fi
-  else
-    "${GH}" pr edit "${pr}" --repo "${repo_url}" --remove-label "${CI_HOST}-Running" --add-label "${CI_HOST}-Failed"
+    #############################################################
+    # loop over every yaml file in ${WF_ROOT_DIR}/ci/experiments
+    # and create an run directory for each one for this PR loop
+    #############################################################
+    for yaml_config in "${WF_ROOT_DIR}/ci/experiments/"*.yaml; do
+      pslot=$(basename "${yaml_config}" .yaml) || true
+      export pslot
+      "${WF_ROOT_DIR}/ci/scripts/create_experiment.py" --yaml "${WF_ROOT_DIR}/ci/experiments/${pslot}.yaml"
+      ci_status=$?
+      if [[ ${ci_status} -eq 0 ]]; then
+        {
+          echo "Created experiment"
+          echo "Experiment setup: Completed at $(date) for expirment ${pslot}" || true
+        } >> "${GFS_CI_ROOT}/PR/${pr}/output_${id}"
+      else 
+        {
+          echo "Failed on createing experiment ${pslot}"
+          echo "Experiment setup: failed at $(date) for expirment ${pslot}" || true
+        } >> "${GFS_CI_ROOT}/PR/${pr}/output_${id}"
+        "${GH}" pr edit "${pr}" --repo "${repo_url}" --remove-label "${CI_HOST}-Running" --add-label "${CI_HOST}-Failed"
+        exit 1
```

Comment:
Not sure if we want to exit here, as it will prevent the processing of additional PRs.